etl/chart_revision_suggester.py

```diff
@@ -1,11 +1,19 @@
+"""
+Using the <mapping-file> JSON file mapping variable IDs from an old dataset to a new dataset, it creates a bunch of suggested chart revisions in Grapher.
+
+It connects to Grapher based on the environment file found in the project's root directory `path/to/etl/.env`.
```

Comment:
FYI you can use `ENV=.env.staging etl-chart-suggester ...` or `ENV=.env.production etl ...` assuming your staging / prod envs are in file `.env.staging`, ... This seemed less prone to errors than having a single `.env` and modifying its contents.